src/modules/material-requests/controllers/material-requests.controller.ts

```diff
@@ -20,20 +20,15 @@ export class MaterialRequestsController {
 
 	@Get('personal')
 	@ApiOperation({
-		description: 'Get Material Requests endpoint for User.',
+		description: 'Get material requests for the logged in user.',
 	})
-	@RequireAllPermissions(
-		Permission.READ_PERSONAL_APPROVED_MATERIAL_REQUESTS,
-		Permission.MANAGE_ACCOUNT
-	)
+	@RequireAllPermissions(Permission.VIEW_OWN_MATERIAL_REQUESTS, Permission.MANAGE_ACCOUNT)
 	public async getPersonalMaterialRequests(
 		@Query() queryDto: MaterialRequestsQueryDto,
 		@SessionUser() user: SessionUserEntity
 	): Promise<IPagination<MaterialRequest>> {
-		const visits = await this.materialRequestsService.findAll(queryDto, {
+		return await this.materialRequestsService.findAll(queryDto, {
```

Comment:
no need to await the promise you return in an async function. Otherwise you resolve that promise and the async function rewraps it with a promise. Which is unnecessary work.
```suggestion
		return this.materialRequestsService.findAll(queryDto, {
```

Reply:
Adjusted